docs/api/standard_library_types.md

```diff
@@ -151,3 +151,14 @@ m = Model(td='P3DT12H30M5S')
 print(m.model_dump())
 #> {'td': datetime.timedelta(days=3, seconds=45005)}
 ```
+
+## IP Address Types
+
+* `ipaddress.IPv4Address`: Simply uses the type itself for validation by passing the value to `IPv4Address(v)`.
+* `ipaddress.IPv4Interface`: Simply uses the type itself for validation by passing the value to `IPv4Address(v)`.
+* `ipaddress.IPv4Network`: Simply uses the type itself for validation by passing the value to `IPv4Network(v)`.
+* `ipaddress.IPv6Address`: Simply uses the type itself for validation by passing the value to `IPv6Address(v)`.
+* `ipaddress.IPv6Interface`: Simply uses the type itself for validation by passing the value to `IPv6Interface(v)`.
+* `ipaddress.IPv6Network`: Simply uses the type itself for validation by passing the value to `IPv6Network(v)`.
```

Comment:
Not important... But do we need the word "simply"? 👀

Reply:
Just copied them from the old file. I've Removed them.